src/components/record/RecordForm.jsx

```diff
@@ -116,16 +116,18 @@ class RecordForm extends React.Component {
       <>
         <PromiseTrackingMask area="sform" />
         {!!this.props.form && (
-          <SForms
-            ref={this.refForm}
-            form={this.props.form}
-            formData={this.props.record.question}
-            options={options}
-            fetchTypeAheadValues={this.fetchTypeAheadValues}
-            isFormValid={this.props.isFormValid}
-            enableForwardSkip={true}
-            loader={<Loader />}
-          />
+          <IntlContextProvider locale={options.intl.locale}>
```

Comment:
As mentioned in https://github.com/kbss-cvut/s-forms/issues/193, there are 2 ways of providing intl in SForms now, which should be refactored.